PKGBUILD

```diff
@@ -23,6 +23,7 @@ build() {
 package() {
   cd scx-${pkgver}
   meson install -C build --destdir "${pkgdir}"
+  install -Dm644 "LICENSE" "${pkgdir}/usr/share/licenses/$pkgname/LICENSE"
```

Comment:
Yeah so the thing I'm not 100% sure about is whether this line is necessary per https://wiki.archlinux.org/title/PKGBUILD#license:

>For common licenses (like 'GPL-3-or-later'), package [licenses (https://archlinux.org/packages/name=licenses) delivers all the corresponding files. The package is installed by default, as it is a dependency of [base](https://archlinux.org/packages/?name=base) [meta package](https://wiki.archlinux.org/title/Meta_package), and the files may be found in /usr/share/licenses/common/. Simply refer to the license using its SPDX license identifier from the [list of SPDX identifiers](https://spdx.org/licenses/).

Did you try installing this package without this line? The way that paragraph is phrased above makes it sound like `makepkg` will install the GPL-v2 license automatically, but looking at the pacman source I don't see it.

Reply:
This package you mentioned contains license content that you can use in your project: https://archlinux.org/packages/core/any/licenses/

```
install -Dm644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
```
This means that there must be a license in the program sources and it is put together with the code into one package. This license is taken directly from here: https://github.com/sched-ext/scx/blob/main/LICENSE

Reply:
An alternative is to put a line like this in the meson.build file:

```
install_data('LICENSE', install_dir : get_option('datadir') / 'licenses' / 'scx-scheds')
```
Then meson will automatically recognize the path for any distribution that contains the "licenses" subdirectory, and meson will install the manifest directly from the sources, without additional entries in PKGBUILD. In the case of spec it's not so easy, because you have to list all the files, nevertheless the path should detect without any problem. However, it seems to me that this pull request solution will be more suitable.

Reply:
Ahh ok, so the package really does nothing other than provide the files for the license. That's kind of odd, but OK. This LGTM, thanks Piotr. Will stamp but let @htejun merge as he's the package maintainer

Reply:
Yes, this package only provides license files. It has no other role.